packages/maskbook/src/plugins/GoodGhosting/UI/GameAction.tsx

```diff
@@ -55,7 +55,7 @@ export function GameAction(props: GameActionProps) {
             await action()
             props.info.refresh()
         } catch (error) {
```

Comment:
plz check error type

we currently use typescript 4.3, but i hope you don't prevent we upgrade to typescript 4.4

see https://devblogs.microsoft.com/typescript/announcing-typescript-4-4-beta/#use-unknown-catch-variables

Reply:
https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards

```ts
function isTranscationError(error: unknown): error is TranscationError {
     // check logic
}
```

```ts
// ...
} catch (error) {
    if (isTranscationError(error)) {
        // your business
    }
}
// ...
```

Reply:
Done!
Thanks for the tips :+1: